app/assets/javascripts/pretty-text/engines/discourse-markdown/quotes.js

```diff
@@ -36,6 +36,14 @@ const rule = {
           full = true;
           continue;
         }
+
+        // if we have the additional attribute of username: because we are prioritizing full name
+        // then assign the name to be the displayName
+        if (split[i].indexOf("username:") === 0) {
+          displayName = split[0];
+          username = split[i].substr(9);
+          continue;
```

Comment:
```suggestion
        // if we have the additional attribute of username: because we are prioritizing full name
        // then assign the name to be the displayName
        if (split[i].indexOf("username:") === 0) {
          displayName = split[0];
          username = split[i].substr(9);
          continue;
```